app/templates/cadastro-esportes.html

```diff
@@ -8,7 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="{% static 'css/navbar-cadastro.css' %}">
     <link rel="stylesheet" href="{% static 'css/base.css' %}">
-    <link rel="stylesheet" href="{% static 'css/esportes.css' %}">
+    <link rel="stylesheet" href="{% static 'css/cadastro-geral.css' %}">
```

Comment:
Let's start following the pattern of always using everything in **English**

Vamos começar a seguir o padrão de sempre usar tudo em inglês

Reply:
Either you do everything in Portuguese or everything in English, no mixing languages

Ou você faz tudo em português ou tudo em inglês, sem misturar idiomas